frontend/packages/kubevirt-plugin/integration-tests/tests/vm.overview.scenario.ts

```diff
@@ -0,0 +1,123 @@
+import { getVmManifest, basicVmConfig } from './utils/mocks';
+import { exposeServices } from './utils/utils';
+import { VirtualMachine } from './models/virtualMachine';
+import { TABS, DASH, VM_BOOTUP_TIMEOUT_SECS } from './utils/consts';
+import { NodePortService } from './utils/types';
+
+describe('Test vm overview', () => {
+  const vmName = `vm-${testName}`;
+  const cloudInit = `#cloud-config\nuser: cloud-user\npassword: atomic\nchpasswd: {expire: False}`;
+  const serviceCommon = { name: vmName, kind: 'vm', type: 'NodePort', namespace: testName };
+  const testVm = getVmManifest('Container', testName, vmName, cloudInit);
+  const vm = new VirtualMachine(testVm.metadata);
+  const nodePortServices = new Set<NodePortService>();
+  nodePortServices.add({
+    ...serviceCommon,
+    exposeName: `${vmName}-service-ssh`,
+    port: '22',
+    targetPort: '20022',
+  });
+  nodePortServices.add({
+    ...serviceCommon,
+    exposeName: `${vmName}-service-smtp`,
+    port: '25',
+    targetPort: '20025',
+  });
+  nodePortServices.add({
+    ...serviceCommon,
+    exposeName: `${vmName}-service-http`,
+    port: '80',
+    targetPort: '20080',
+  });
+
+  beforeAll(() => {
+    createResource(testVm);
+    exposeServices(nodePortServices);
+  });
+
+  afterAll(() => {
+    deleteResource(testVm);
+  });
+
+  beforeEach(async () => {
+    await vm.navigateToTab(TABS.OVERVIEW);
+    await isLoaded();
+  });
+
+  it('Check vm details in overview when vm is off', async () => {
+    const expectation = {
+      name: vmName,
+      description: testName,
+      os: basicVmConfig.operatingSystem,
+      profile: basicVmConfig.workloadProfile,
+      template: 'Template\nTrhel7-desktop-tiny',
+      bootOrder: ['rootdisk', 'nic0', 'cloudinitdisk'],
+      flavor: basicVmConfig.flavor,
+      ip: DASH,
+      pod: DASH,
+      node: DASH,
+      offIconPresent: true,
+    };
+
+    const found = {
+      name: await resourceTitle.getText(),
+      description: await vmView.vmDetailDesc(testName, vmName).getText(),
+      os: await vmView.vmDetailOS(testName, vmName).getText(),
+      profile: await vmView.vmDetailWorkloadProfile(testName, vmName).getText(),
+      template: await vmView.vmDetailTemplate(testName, vmName).getText(),
+      bootOrder: await vmView.vmDetailBootOrder(testName, vmName).getText(),
+      flavor: await vmView.vmDetailFlavor(testName, vmName).getText(),
+      ip: await vmView.vmDetailIP(testName, vmName).getText(),
+      pod: await vmView.vmDetailPod(testName, vmName).getText(),
+      node: await vmView.vmDetailNode(testName, vmName).getText(),
+      offIconPresent: await vmView.statusIcon(vmView.statusIcons.off).isPresent(),
+    };
+
+    if (!_.isEqual(found, expectation)) {
+      // eslint-disable-next-line no-console
+      console.error(`Expected:\n${JSON.stringify(expectation)},\nGot:\n${JSON.stringify(found)}`);
+    }
+    expect(_.isEqual(found, expectation)).toBe(true);
```

Comment:
can we reuse the result of the last isEqual call or just call fail inside the if?